corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -228,6 +227,29 @@ def toggle_enabled(request, toggle_or_toggle_name):
     return _toggle_enabled(corehq.toggles, request, toggle_or_toggle_name)
 
 
+@register.simple_tag
+def toggle_js_url(domain, username):
+    return '{url}?username={username}&cachebuster={domain_cb}-{user_cb}'.format(
+        url=reverse('toggles_js', args=[domain]),
+        username=username,
+        domain_cb=toggle_js_domain_cachebuster(domain),
+        user_cb=toggle_js_user_cachebuster(username),
+    )
+
+
+@quickcache(['domain'], timeout=30 * 24 * 60 * 60)
+def toggle_js_domain_cachebuster(domain):
+    # to get fresh cachebusters on the next deploy
+    # change the date below (output from *nix `date` command)
+    #   Thu Mar  3 16:21:30 EST 2016
```

Comment:
It'll only change when the code of this particular function (or the other cachebuster function) changes. And I think we more or less do want that. Anytime we're fiddling with how cachebusting works, we'll probably want everyone to be on whatever the new system is—and recomputing everyone's toggles json is not an extraordinarily intensive computation.